CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
   `#[display(fmt = "...", ("<expr>"),*)]`, and `#[display(bound(<bound>))]`
   instead of `#[display(bound = "<bound>")]`. So without the double quotes
   around the expressions and bounds.
+- The `Display` derive (and other `fmt`-like ones) now delegates to the inner type
```

Comment:
I think we should explicitely call out Debug too. In the docs this is listed as a separate derive.

Reply:
@JelteF since this is a CHANGELOG, and before this release there was no clear separation, I've thought that it's enough "and other `fmt`-like ones" to consider `Debug` too. But it's not a problem to mention it more explicitly.